test/function/param.cpp

```diff
@@ -628,4 +628,80 @@ namespace {
 
     stumpless_free_all(  );
   }
+
+  TEST( ParamFromStringTest, New ) {
+    struct stumpless_param *param;
+    const char *string = "test-param-name=\"test-param-value\"";
+
+    size_t name_length = strlen("test-param-name");
+    size_t value_length = strlen("test-param-value");
+
+    param = stumpless_new_param_from_string( string );
```

Comment:
The use of a variable named `string` is confusing the header check tool, which is why you see the static analysis CI failure. It believes it is a reference to the `string` type in C++. In this case, I think it would be better to rename this variable so that there isn't a name re-use, perhaps to something like `param_string`.